src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceConnectionService.php

```diff
@@ -177,4 +179,42 @@ private function assertValidMetadata(ManageEntity $entity): MetaData
         }
         return $metadata;
     }
+
+    /**
+     * @return array<string, IdentityProvider>
+     */
+    private function getRemainingIdpConnections(InstitutionId $institutionId)
+    {
+        $items = $this->idpService->findInstitutionIdps($institutionId);
+        $indexed = [];
+        foreach ($items->institutionEntities() as $idp) {
+            $indexed[$idp->getEntityId()] = $idp;
+        }
+        return $indexed;
+    }
+
+    /**
+     * @param array<string, IdentityProvider> $testIdpsIndexed
+     * @return array<string, IdentityProvider>
+     */
+    private function gatherConnectedIdps(ManageEntity $entity, array $testIdpsIndexed): array
```

Comment:
What is the diff between get and gather?

Reply:
I found gather slightly more descriptive at the time. In the mean time this method has been made redundant. And is no longer part of the coebase